packages/mezzio-toggle/src/ToggleConfigProvider.php

```diff
@@ -26,7 +26,7 @@
 use Pheature\Crud\Psr7\Toggle\GetFeatures;
 use Pheature\Crud\Psr7\Toggle\PatchFeature;
 use Pheature\Crud\Psr7\Toggle\PostFeature;
-use Pheature\Crud\Toggle\Handler\AddStrategy;
+use Pheature\Crud\Toggle\Handler\SetStrategy;
```

Comment:
Renamed the entire add strategy flow to set strategy because its behavior complains both add and replace actions.